tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
Hi Bjoern,

I’m working on yet another tool that requires Image metadata in Galaxy.  What do you think is the best approach for getting this into Galaxy?  I’d rather not get involved if others have this on their plate, but I’m willing to work on this.  But only if whatever I do will be accepted by the dev team (i.e., I don’t want to waste time on something that won’t be accepted).  So do you know if this is already on someone else’s plate?  Is there a Trello card that outlines the direction the dev team wants to go on this?

Thanks!

> On Aug 4, 2015, at 10:49 AM, Björn Grüning <notifications@github.com <mailto:notifications@github.com>> wrote:
> Ok, let's work on proper image metadata after the first workflow is ready for prime time.
>